manifest.md

```diff
@@ -143,6 +151,78 @@ Notice that the `mediaType` is subject to the usage or context, while the digest
 }
 ```
 
+## Guidelines for Artifact Usage
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type.
+For portability, if the artifact does not have a content for the config blob, an empty json object, `{}`, SHOULD be pushed for the blob content.
+If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
+The suggested content for an unused layer is the empty json object, `{}`.
+When the content of the config or layer blob uses the empty json object, the `mediaType` for its descriptor SHOULD be set to the type of the artifact rather than the type of the blob.
+
+Here is an example manifest for a typical artifact:
+
+```jsonc
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example.config.v1+json",
+    "digest": "sha256:5891b5b522d5df086d0ff0b110fbd9d21bb4fc7163af34d08286a2e846f6be03",
+    "size": 123
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.example.data.v1.tar+gzip",
+      "digest": "sha256:e258d248fda94c63753607f7c4494ee0fcbe92f1a76bfdac795c9d84101eb317",
+      "size": 1234
+    }
+  ]
+}
+```
+
+Here is an example manifest for a simple artifact without content in the config:
+
+```jsonc
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example+type",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.example+type",
+      "digest": "sha256:e258d248fda94c63753607f7c4494ee0fcbe92f1a76bfdac795c9d84101eb317",
+      "size": 1234
+    }
+  ]
+}
+```
+
+Here is an example minimal manifest for an artifact, with the content of both blobs set to `{}`:
+
+```jsonc
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example+type",
```

Comment:
`+oci` was agreed in the meeting. Please update the example to reflect guidance for single blob artifacts that want to represent the artifactType to align with the mediaType.

Reply:
/cc @vbatts

Reply:
I'd updated with the requested changes.